### ISSUE TITLE
Gruppenzusammensetzung/Anforderungen HSR

### ISSUE THREAD

**Issue by cbreu** [CASFEE16/projekt2#3]
Soll ich mit Tobias/Markus mal abklären, was die Schule zu unserem Plan, das Projekt 2 zu dritt mit Mike zusammen zu machen, meint?

**Comment by patlau** [comment]
Ja, gerne oder wir schauen es heute Abend an, ist ja schon bald soweit ;-)

**Comment by cbreu** [comment]
Das machen wir so...
